tests/test_spam_admin.py

```diff
@@ -0,0 +1,440 @@
+
+        assert_equal(comment_spam.spam_status, Comment.SPAM)
+
+
+    ###############################  TEST VIEW   #############################################
+    @httpretty.activate
+    def test_delete_if_marked_as_spam(self):
+
+
+        comment = self._add_comment(
+            self.project, auth=self.project.creator.auth,
+        )
+        comment.mark_as_possible_spam(save=True)
+
+        if self.spam_assassin_active:
+            self.app.post_json(
+                '/api/v1/spam_admin/mark_comment_as_spam/',
+                {
+                    'cid':comment._id
+                },
+                auth=self.spam_admin.auth
+            )
+            comment.reload()
+            assert_equal(comment.spam_status, Comment.SPAM)
+            assert_true(comment.is_deleted)
+        else:
+            resp = self.app.post_json(
+                '/api/v1/spam_admin/mark_comment_as_spam/',
+                {
+                    'cid': comment._id
+                },
+                auth=self.spam_admin.auth,
+                expect_errors=True
+            )
+            comment.reload()
+            assert_equal(resp.status_code, 400)
+            assert_equal(comment.spam_status, Comment.POSSIBLE_SPAM)
+            assert_false(comment.is_deleted)
+
+    @httpretty.activate
+    def test_marked_as_ham(self):
+        comment = self._add_comment(
+            self.project, auth=self.project.creator.auth,
+        )
+        comment.mark_as_possible_spam(save=True)
+        if self.spam_assassin_active:
+            self.app.post_json(
+                '/api/v1/spam_admin/mark_comment_as_ham/',
+                {
+                    'cid':comment._id
+                },
+                auth=self.spam_admin.auth
+            )
+            comment.reload()
+            assert_equal(comment.spam_status, Comment.HAM)
+        else:
+            resp = self.app.post_json(
+                        '/api/v1/spam_admin/mark_comment_as_ham/',
+                        {
+                            'cid':comment._id
+                        },
+                        auth=self.spam_admin.auth,
+                        expect_errors=True
+            )
+            comment.reload()
+            assert_equal(resp.status_code, 400)
+            assert_equal(comment.spam_status, Comment.POSSIBLE_SPAM)
+            assert_false(comment.is_deleted)
+
+    @httpretty.activate
+    def test_list_possible_spam_comments(self):
+        for i in range(5):
+            self._add_comment(
+                self.project, content=self.GTUBE,auth=self.project.creator.auth,
+            )
+            self._add_comment(
+                self.project, auth=self.project.creator.auth,
+            )
+
+        ret=self.app.get(
+            '/api/v1/spam_admin/list_comments/3/',
+            auth=self.spam_admin.auth
+        )
+
+
+        if self.spam_assassin_active:
+            #todo: stop comments from earlier tests from saving
```

Comment:
Can you get the number of current comments in the list at the beginning of the test and assert the number of added comments from the difference?